.buildkite/ironbank-validation.yml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  # Other deps
+  ASDF_MAGE_VERSION: 1.15.0
+
+steps:
+  - label: ":package: :test_tube: Package and Validate Iron Bank for x-pack/heartbeat"
+    command: |
+      set -euo pipefail
+      export BEAT_VERSION=$$(make get-version)
+      echo "~~~ Using BEAT_VERSION: $$BEAT_VERSION"
+      cd x-pack/heartbeat
+      echo "--- Package x-pack/heartbeat for the artifacts consumed by the Iron Bank docker context"
```

Comment:
these are [separate stages in Jenkins](https://github.com/elastic/beats/blob/079395b7d800cd21d4297f98ca9be2c85b73f5e1/.ci/ironbank-validation.groovy#L51) but with the [advanced output logging features](https://buildkite.com/docs/pipelines/managing-log-output#collapsing-output) of Buildkite I don't think we should waste vm resources, possibility of vm capacity issues etc. to separate them in more steps.